Riot/Modules/TabBar/TabBarCoordinator.swift

```diff
@@ -73,6 +76,10 @@ final class TabBarCoordinator: NSObject, TabBarCoordinatorType {
         self.activityIndicatorPresenter = ActivityIndicatorPresenter()
     }
     
+    deinit {
+        activities.cancelAll()
```

Comment:
What's the difference between `cancel` and `complete` in this context?

Also, I was under the impression this happens automatically on activity deinit?

Reply:
`complete` should not be public API, only `cancel` is. But this manual cancellation isn't necessary, it does indeed automatically deinit, so removing.